src/CallInvite.tsx

```diff
@@ -55,46 +72,149 @@ export declare interface CallInvite {
    */
 
   /**
-   * MessageReceived event. Raised when {@link (CallMessage:class)} is received.
+   * Accepted event. Raised when the call invite has been accepted.
+   *
    * @example
-   * ```typescript
-   * callInvite.addListener(CallInvite.Event.MessageReceived, (message) => {
-   *    // callMessage received
-   * })
+   * ```ts
+   * voice.on(Voice.Event.CallInvite, (callInvite) => {
+   *   callInvite.on(CallInvite.Event.Accepted, () => {
```

Comment:
Do you want to add `call` here as argument? 
```suggestion
   *   callInvite.on(CallInvite.Event.Accepted, (call) => {
```